src/Views/SuccessView.vala

```diff
@@ -60,8 +60,15 @@ public class Sideload.SuccessView : AbstractView {
             );
         }
 
+        Sideload.Application app = ((Sideload.Application) GLib.Application.get_default ());
+        var file = app.main_window.file;
+        var trash_flatpakref_on_exit = new Gtk.CheckButton.with_label (_("Trash \"%s\"").printf (file.file.get_basename ()));
```

Comment:
I wonder if this should instead be "Move %s to Trash" to mirror the menuitem label we use in Files. Thoughts @cassidyjames ?

```suggestion
        var trash_flatpakref_on_exit = new Gtk.CheckButton.with_label (_("Move “%s” to Trash").printf (file.file.get_basename ()));
```

Reply:
@danrabbit agreed, that sounds better and more consistent.

I think this would be more clear and consistent with out typical style of including the widget type in the variable name:

```suggestion
        var trash_check = new Gtk.CheckButton.with_label (_("Move “%s” to Trash").printf (file.file.get_basename ()));
```